vic/drivers/image/Makefile

```diff
@@ -75,6 +75,7 @@ INCLUDES = -I ${DRIVERPATH}/include \
 
 # Uncomment to include debugging information
 CFLAGS  =  ${INCLUDES} ${NC_CFLAGS}  -ggdb -O0 -Wall -Wextra -std=c99 \
+					 -fopenmp \
```

Comment:
In general, RASM is building vic with a RASM Makefile.  We could change that so it builds a vic library for RASM using the vic build scripts, but I'm not suggesting that.  Ultimately, we just need to modify the RASM build so it builds vic5 properly.  This should not be too hard to do.